atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraTimestampsEteTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.ete;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.awaitility.Awaitility;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.Iterables;
+import com.palantir.atlasdb.ete.cassandra.util.CassandraCommands;
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.atlasdb.todo.ImmutableTodo;
+import com.palantir.atlasdb.todo.Todo;
+import com.palantir.atlasdb.todo.TodoResource;
+import com.palantir.atlasdb.todo.generated.TodoTable;
+
+public class CassandraTimestampsEteTest {
+    private static final Todo TODO = ImmutableTodo.of("todo");
+    private static final Todo TODO_2 = ImmutableTodo.of("todo_two");
+    private static final String CASSANDRA_CONTAINER_NAME = "cassandra";
+    private static final long ID = 1L;
+
+    private TodoResource todoClient = EteSetup.createClientToSingleNode(TodoResource.class);
+
+    @Before
+    public void setup() {
+        todoClient.truncate();
+    }
+
+    @After
+    public void cleanup() {
+        todoClient.truncate();
+    }
+
+    @Test
+    public void timestampsInOneSSTableAreCurrent() throws IOException, InterruptedException {
```

Comment:
Consider adding a comment about the implications if this test fails. Out of curiosity, have you attempted to run it on a pre-3224 branch?

Reply:
Added several comments including an additional assertion that's important. Actually the todo table is thorough (with regards to the general comment I wanted to do one for a conservative table that I thought maybe could be a part 4).